ADotNet.Infrastructure.Build/Program.cs

```diff
@@ -50,19 +50,18 @@ static void Main(string[] args)
 
                         Steps = new List<GithubTask>
                         {
-                            new CheckoutTaskV2
+                            new CheckoutTaskV3
                             {
                                 Name = "Check out"
                             },
 
-                            new SetupDotNetTaskV1
+                            new SetupDotNetTaskV3
                             {
                                 Name = "Setup .Net",
 
-                                TargetDotNetVersion = new TargetDotNetVersion
+                                TargetDotNetVersion = new TargetDotNetVersionV3
                                 {
-                                    DotNetVersion = "7.0.201",
-                                    IncludePrerelease = false
+                                    DotNetVersion = "7.0.201"
```

Comment:
Current version is "7.0.203", is there reasoning to not use 7.0.*

Reply:
I think the correct syntax is 7.0.x
I have used this on my own projects using `ADotNet` and it seems to work, It builds latest non pre release.

Reply:
Thanks Brian your correct my bad 7.0.* is never going to work 👍 @cjdutoit said he received some errors with the latest version so he reverted back to 7.0.201, I have used the syntax 7.0.x as well.